netbox/project-static/styles/overrides/_tabler.scss

```diff
@@ -36,11 +36,55 @@ pre {
 }
 
 table a {
-  // Adjust table anchor link contrast as not enough contrast in dark mode
-  filter: brightness(110%);
+  color: $dark-teal;
```

Comment:
I'm pretty sure we can remove this section; the link color should already be set to `$dark-teal`.